DakotaLambert/VenueVagabondAPI
# List of states from postman, initial model creation

#### Changes Made

1. Migrated all Django models
2. Created project models, migrated
3. Created initial fixtures to seed database/use authentication
4. Created initial `State` VIEWSET 

#### Steps to Review
1. Checkout to this branch locally
   ```
   git fetch --all
   git checkout DL-StateView
   ```

2. Ensure the server and client are both running.

4. Test app functionality.
   
   > When requesting `localhost:8000/states` via PostMan, a response of the US states should be returned.

5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
